README.md

```diff
@@ -87,7 +87,10 @@ TRACE_PORT = "4318"
 configure_tracer(
     enabled=False, # Set to True when TRACE_HOST is configured.
     service_name='SERVICE_NAME',
-    endpoint=f"http://{TRACE_HOST}:{TRACE_PORT}/v1/traces",
+    exporter=Exporter(
```

Comment:
Can we simplify this a bit further when we first look into this? 

```suggestion
    exporter=Exporter(
      host=TRACE_HOST,
      port=TRACE_PORT,
      exporter_type=ExporterType.HTTP,
      scheme: Optional[str] = None,
      path: Optional[str] = None,
    )
```

We can probably just use `http` as the default scheme, and add `/v1/traces` as the path when http is used. We can probably do this to make it easier for people to use the library.

Reply:
Good idea, I simplified it even more so you only need to specify host in most cases.